api/prd/judicial/index.ts

```diff
@@ -36,6 +36,23 @@ export async function searchJudicialUserByPersonalCodes(req: EnhancedRequest, re
   }
 }
 
+export async function searchJudicialUserByIdamId(req: EnhancedRequest, res: Response, next: NextFunction) {
+  const reqBody = req.body;
+  const markupPath: string = `${prdUrl}/refdata/judicial/users`;
+  try {
+    // Judicial User search API version to be used depends upon the config entry FEATURE_JRD_E_LINKS_V2_ENABLED's value
+    req.headers.accept = showFeature(FEATURE_JRD_E_LINKS_V2_ENABLED)
```

Comment:
Good that you are accounting for this, but e links should be on everywhere now

Reply:
removed the feature check and just set to use v2 accept header